docs/illumos-and-smartos-basics/glossary.md

```diff
@@ -0,0 +1,367 @@
+specifications that software programs can follow to communicate with each other.
+" - [wikipedia](https://en.wikipedia.org/wiki/Api)
+
+## CDN
+
+Content Delivery Network  
+"A system of computers containing copies of data placed at various nodes of a
+network. When properly designed and implemented, a CDN can improve access to the
+data it caches by increasing access bandwidth and redundancy and reducing access
+latency." - [wikipedia](https://en.wikipedia.org/wiki/Content_delivery_network)
+
+## Clone
+
+A writable [snapshot](glossary.md#snapshot) of a file system.
+
+## Cloud
+
+"... a style of computing in which dynamically scalable and often virtualized
+resources are provided as a service over the internet." -
+[wikipedia](https://en.wikipedia.org/wiki/Cloud_computing)
+
+The [National Institute of Standards and Technology](https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-145.pdf)
+(pdf): "Cloud computing is a model for enabling ubiquitous, convenient,
+on-demand network access to a shared pool of configurable computing resources
+(e.g., networks, servers, storage, applications, and services) that can be
+rapidly provisioned and released with minimal management effort or service
+provider interaction."
+
+## Colocation
+
+"A type of data center... Colocation allows multiple customers to locate
+network, server, and storage gear - and connect them to a variety of
+telecommunications and network service providers - with a minimum of cost and
+complexity." - [wikipedia](https://en.wikipedia.org/wiki/Colocation_centre)
+
+## Compute node
+
+Compute nodes are the physical servers in a [datacenter](glossary.md#datacenter)
+that are managed by a [head node](glossary.md#head-node).
+
+## CRM
+
+Customer Relationship Management  
+"... a widely-implemented strategy for managing a company’s interactions with
+customers, clients and sales prospects. It involves using technology to
+organize, automate, and synchronize business processes---principally sales
+activities, but also those for marketing, customer service, and technical
+support." -
+[wikipedia](https://en.wikipedia.org/wiki/Customer_Relationship_Management)
+
+## Customer portal
+
+The customer portal is a reference portal that consumers can use to manage their
+[SmartMachine](glossary.md#smartmachine).
+
+## Datacenter
+
+A datacenter is both a physical and virtual location that connects to the
+Internet, and holds a [head node](glossary.md#head-node) and one or more
+[compute nodes](glossary.md#compute-node). A datacenter may have several head
+nodes. In this case each head node has its own name to differentiate between
+each of their instances.
+
+## Dataset
+
+A dataset describes the software that's going to end up on a newly provisioned
+machine. A dataset is sometimes called an image. All machines provisioned from
+the same dataset have the same software. On Virtual Machines, the dataset also
+includes the operating system and the disk that contains it.
+
+## DIRT
```

Comment:
I'm not sure this entry is useful.

Reply:
Since we are not sure if is useful, I removed the term. I think that is better starting only having what we think that is useful.